Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
Hmm, that could work. We'd want to rename the variable from HEROKU to HEROKU_SINGLE_DYNO or some such. Only thing I'm not sure of: will Heroku by default provision 1 web and 1 worker (which will exceed free capacity during the month)? I'll do some tests.